taskcluster/ci/config.yml

```diff
@@ -14,11 +14,11 @@ taskgraph:
 
 workers:
     aliases:
-        b-linux:
+        bitrise:
             provisioner: 'mobile-{level}'
             implementation: docker-worker
             os: linux
-            worker-type: 'b-linux'
+            worker-type: 'bitrise'
```

Comment:
I created this new worker-type. It's a type defined for taskcluster. It currently maps our capacity in bitrise https://hg.mozilla.org/ci/ci-configuration/rev/39f5c268f49092e33278ee1ddd2bdf0cfb09db73#l1.33. If I kept using `b-linux`, we would end up with 10 taskcluster workers all delegating work to bitrise as the same time. This means 7 of these workers would be polling Bitrise while knowing Bitrise not doing anything for them.